docs/10.0/guides/advanced-topics/performance.md

```diff
@@ -72,26 +72,6 @@ See the [batch operations](@/guides/advanced-topics/batch-operations.md) page to
 
 ## Use modules
 
-If you need only a few parts of Handsontable, you can think of importing them as **modules**. Eventually, this can lead to lowering the bundle size. This can be done in several steps:
-
-- import the base
-- import the module you want to use and its registering method
-- register it
-- use it
-
-The following example shows you how to import and register the `ContextMenu`
-
-```js
-import Handsontable from 'handsontable/base';
-import { registerPlugin, ContextMenu } from 'handsontable/plugins';
-
-registerPlugin(ContextMenu);
-
-// switch the context menu on
-new Handsontable(container, {
-  contextMenu: true,
-  // other settings
-});
-```
-
-You can also optimize the use of **moment.js**. To find out more about this topic, see the [modules page](@/guides/building-and-testing/modules.md).
+To make your grid lighter:
+- [Use Handsontable modules &#8594;](@/guides/building-and-testing/modules.md#using-modules)
```

Comment:
Both of those links are redirecting me to the same place in the docs
![Zrzut ekranu 2022-01-5 o 11 13 31](https://user-images.githubusercontent.com/10757813/148200624-2c18ab64-e145-403d-a187-4a1b709977f3.png)

here 

![Zrzut ekranu 2022-01-5 o 11 14 40](https://user-images.githubusercontent.com/10757813/148200673-416c1046-47af-4860-aff0-4b3bc283c705.png)

Reply:
@AMBudnik - you probably opened those in new tabs? That's an issue with bookmark links not opening properly in a new tab when running the docs on local server. It works in the Live environment, though.

To see this, for example:
1. Go here: https://handsontable.com/docs/api/options/#allowinvalid
2. Open the **Autocomplete strict mode** link in a new tab (it works fine).
3. Build docs locally and go here: http://localhost:8080/docs/next/api/options/#allowinvalid
4. Open the **Autocomplete strict mode** link in a new tab (it opens on the main **Autocomplete cell type** page - the bookmark doesn't work.

Reply:
ok, got it 👍

Reply:
I agree that we could make it slightly clearer by extending the text

> To make your grid lighter

Into something with more context.

> To reduce the bundle size and JavaScript parsing time, we advise to import only the [modules](link) of Handsontable that you actually use, instead of the complete package.

-----

I think that the `moment.js` tip should be moved from the "Modules" page to the "Date cell type" page: https://handsontable.com/docs/date-cell-type/ @budnix do you agree?